README.md

```diff
@@ -12,27 +12,93 @@ As mentioned in our [FAQ](https://www.infracost.io/docs/faq), no cloud credentia
 
 * [Usage methods](#usage-methods)
   * [Docker image](#option-1-docker-image)
-    * [Environment variables](#environment-variables)
-  * [Plan JSON API](#option-2-plan-json-api)
+  * [Install in Docker](#option-2-install-in-docker)
 * [Contributing](#contributing)
 
 # Usage methods
 
-There are two methods of integrating Infracost with Atlantis:
-1. Use a custom Docker image that [extends](https://www.runatlantis.io/docs/deployment.html#customization) Atlantis' `latest` image to add Infracost (latest release, v0.9.16). This is the recommended method.
+There are three methods of integrating Infracost with Atlantis:
+1. Use a custom Docker image that [extends](https://www.runatlantis.io/docs/deployment.html#customization) an Atlantis image to add Infracost (latest release, v0.9.16). This is the recommended method.
 
-2. Send the `$PLANFILE` from Atlantis to the Infracost [plan JSON API](https://www.infracost.io/docs/integrations/infracost_api) with `curl`. Whilst this API deletes files from the server after they are processed, it is a good security practice to remove secrets from the file before sending it to the API. For example, AWS provides [a grep command](https://gist.github.com/alikhajeh1/f2c3f607c44dabc70c73e04d47bb1307) that can be used to do this.
+2. Use a pre-workflow hook to dynamically install the Infracost CLI on a running Atlantis server. 
+
+3. Send the `$PLANFILE` from Atlantis to the Infracost [plan JSON API](https://www.infracost.io/docs/integrations/infracost_api) with `curl`. Whilst this API deletes files from the server after they are processed, it is a good security practice to remove secrets from the file before sending it to the API. For example, AWS provides [a grep command](https://gist.github.com/alikhajeh1/f2c3f607c44dabc70c73e04d47bb1307) that can be used to do this.
 
 ## Option 1: Docker image
 
-This method runs `infracost diff` using the `$PLANFILE` that Atlantis generates. The following steps describe how you can use this method:
+[This Docker image](https://hub.docker.com/repository/docker/infracost/infracost-atlantis/) extends the Atlantis image by adding the Infracost CLI. If you already use a custom Docker image for Atlantis, copy the `RUN` commands from [this Dockerfile](https://github.com/infracost/infracost-atlantis/blob/master/Dockerfile) into your Dockerfile.
+
+The `infracost-atlantis` image is maintained with tags for the latest three 0.x versions of Atlantis. For example, if the latest 0.x versions of Atlantis are v0.18.1, v0.17.6, and v0.16.1, the following images will be published/updated when Infracost v0.9.17 is released:
+ 
+- infracost-atlantis:atlantis0.16-infracost0.9 with Atlantis v0.18.1 and Infracost v0.9.17 
+- infracost-atlantis:atlantis0.17-infracost0.9 with Atlantis v0.17.6 and Infracost v0.9.17
+- infracost-atlantis:atlantis0.18-infracost0.9 with Atlantis v0.16.1 and Infracost v0.9.17
+- infracost-atlantis:latest with Atlantis v0.18.1 and Infracost v0.9.17
```

Comment:
Removing mention of Atlantis v0.16 release as it's not available among Atlantis GitHub tagged packages: https://github.com/runatlantis/atlantis/pkgs/container/atlantis/versions

```suggestion
The `infracost-atlantis` image is maintained with tags for the latest three 0.x versions of Atlantis. For example, if the latest 0.x versions of Atlantis are v0.18.1 and v0.17.6, the following images will be published/updated when Infracost v0.9.17 is released:
 
- infracost-atlantis:atlantis0.18-infracost0.9 with Atlantis v0.18.1 and Infracost v0.9.17 
- infracost-atlantis:atlantis0.17-infracost0.9 with Atlantis v0.17.6 and Infracost v0.9.17
- infracost-atlantis:latest with Atlantis v0.18.1 and Infracost v0.9.17
```